themes/digital.gov/src/scss/new/_images.scss

```
@@ -43,3 +43,7 @@
   @include u-float('left');
   @include u-width('card-lg');
 }
+
+p img[alt] {
```

Let's add this as a more global style.

```suggestion
img[alt] {
```